html/changelog.html

```diff
@@ -58,6 +58,23 @@
 <!-- DO NOT REMOVE OR MOVE THIS COMMENT! THIS MUST BE THE LAST NON-EMPTY LINE BEFORE THE LOGJS #ADDTOCHANGELOGMARKER# -->
 
 <div class='commit sansserif'>
+	<h2 class='date'>17 July 2015</h2>
+	<h3 class='author'>DerptheStewpidGoat updated:</h3>
+	<ul class='changes bgimages16'>
+		<li class='rscadd'>New Spetsnaz mob (admin-spawn only)</li>
+		<li class='rscadd'>Secways now make noise whenever they move.</li>
+		<li class='tweak'>Very minor adjustments to the sprites of the camo boots.</li>
+		<li class='tweak'>The gas mask is now using the older sprites.</li>
+	</ul>
+	
```

Comment:
Please don't do this, since this isn't one big commit, the change made by @nalarmg is not included in this pull, so please change it to 

```
 <div class='commit sansserif'>
	<h2 class='date'>17 July 2015</h2>
	<h3 class='author'>DerptheStewpidGoat updated:</h3>
	<ul class='changes bgimages16'>
		<li class='rscadd'>New Spetsnaz mob (admin-spawn only)</li>
		<li class='rscadd'>Secways now make noise whenever they move.</li>
		<li class='tweak'>Very minor adjustments to the sprites of the camo boots.</li>
		<li class='tweak'>The gas mask is now using the older sprites.</li>
	</ul>
</div>
```